### PR TITLE
Enabling caching on view API

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Upgrades packages
- [x] Enables caching and scheduling tasks on the view microservice

**_HOW DO I TEST OUT THIS PR?_**
Follow instructions in the [readme.md file](https://github.com/airqo-platform/AirQo-api/tree/staging/src/view#exposing-the-api)

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
[API Docs](https://platform.airqo.net/api/v1/view/docs/index.html)

### CODE REVIEW

File: src/view/src/main/java/airqo/ViewApplication.java [hunk 1]
```diff
@@ -2,12 +2,16 @@
...
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 import javax.annotation.PostConstruct;
 import java.util.Date;
 import java.util.TimeZone;
 
 @SpringBootApplication
+@EnableScheduling
+@EnableCaching
```
Comment (NoahNsimbe) [inline src/view/src/main/java/airqo/ViewApplication.java:14]: Enabling caching and scheduling

File: src/view/pom.xml [hunk 2]
```diff
@@ -71,14 +71,14 @@
...
-        <version>31.0.1-jre</version>
+        <version>31.1-jre</version>
       </dependency>
 
       <!-- Sentry -->
       <dependency>
         <groupId>io.sentry</groupId>
         <artifactId>sentry-spring-boot-starter</artifactId>
-        <version>5.6.1</version>
+        <version>5.7.0</version>
```
Comment (NoahNsimbe) [inline src/view/pom.xml:81]: upgrading packages

**Comment by NoahNsimbe** [comment]
@Baalmart @123MwanjeMike any reservations??

**Comment by 123MwanjeMike** [comment]
> @Baalmart @123MwanjeMike any reservations??

None from my end @NoahNsimbe

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Pull request merged**